decisions4s-core/src/main/scala/decisions4s/internal/DiagnosticsPrinter.scala

```diff
@@ -30,7 +30,9 @@ class DiagnosticsPrinter[Input[_[_]], Output[_[_]], Out](r: EvalResult[Input, Ou
     val fieldSatisfaction: Vector[Boolean] = HKDUtils.collectFields(rr.details)
     val sign                               = if (rr.evaluationResult.isDefined) then "✓" else "✗"
     val output                             = rr.evaluationResult.map(x => s"== ${renderOutput(x)}").getOrElse("== ✗")
-    s"""Rule $idx [$sign]:
+    val annotation                         = table.rules(idx).annotation.getOrElse("")
+
+    s"""Rule $idx [$sign] $annotation:
```

Comment:
Let’s do after the colon
```suggestion
    s"""Rule $idx [$sign]: $annotation
```

Reply:
done :+1: